app/services/catalog/submit_order.rb

```diff
@@ -22,9 +22,11 @@ def process
     private
 
     def submit_order_item(item)
-      TopologicalInventory.call do |api_instance|
-        result = api_instance.order_service_plan(item.service_plan_ref, parameters(item))
-        update_item(item, result)
+      ManageIQ::API::Common::Request.with_request(item.context) do
```

Comment:
@lindgrenj6 
https://github.com/ManageIQ/catalog-api/blob/2643f9a3749f2a80a570945f91f412e208dc5ce3/spec/services/catalog/add_to_order_spec.rb#L61
Don't you need the transform here

Reply:
ah yep thats it, that is what was causing the test failure I was seeing. Updated.